packages/openneuro-app/src/scripts/refactor_2021/search/inputs/admin-allDatasets-toggle.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, useContext } from 'react'
+import { SearchParamsCtx } from '../search-params-ctx'
+import { Button } from '@openneuro/components/button'
+import styled from '@emotion/styled'
+
+const StyledButton = styled(Button)`
+  width: 100%;
+  padding: 6px 10px;
+  margin-bottom: 20px;
+`
```

Comment:
```suggestion
&.active{
    background: #fff5f3;
    border-color: #e68383;
    color: #710000;
    }
`
```

Reply:
Button Component will need an active class. Unsure if emotion uses the & to concat classNames.